src/lib/component/partial/HandlerDetailsContainer/HandlerDetailsConfiguration.js

```diff
@@ -203,11 +222,16 @@ HandlerDetailsConfiguration.fragments = {
       timeout
       handlers {
         name
+        namespace
       }
       socket {
         port
         host
       }
+      mutator {
```

Comment:
For some the addition of the `mutator` field is throwing an error on the API, when navigating to a Handler Details View. 😕 
<img width="1680" alt="Screen Shot 2019-06-25 at 1 58 59 PM" src="https://user-images.githubusercontent.com/3856248/60133007-7c981e80-9751-11e9-803c-10971d22b038.png">

Reply:
it looks like the API can't handle `null` values for the `mutator` field. When a `Handler.mutator` value is defined, it works.

Reply:
Addressed in https://github.com/sensu/sensu-go/pull/3090. Thank you!